src/main/java/io/confluent/connect/jdbc/source/JdbcSourceConnectorConfig.java

```diff
@@ -330,6 +336,28 @@
       + "  In most cases it only makes sense to have either TABLE or VIEW.";
   private static final String TABLE_TYPE_DISPLAY = "Table Types";
 
+  public static final String TRANSACTION_ISOLATION_MODE_DEFAULT =
+          TransactionIsolationMode.DEFAULT.name();
+  public static final String TRANSACTION_ISOLATION_MODE_CONFIG = "transaction.isolation.mode";
+  private static final String TRANSACTION_ISOLATION_MODE_DOC =
+          "Mode to control which transaction isolation level is used when running queries "
+                  + "against the database. By default no explicit transaction isolation"
+                  + "mode is set. SQL_SERVER_SNAPSHOT_ISOLATION_MODE will only work"
+                  + "against a connector configured to write to Sql Server. "
+                  + " Options include:\n"
+                  + "  * DEFAULT\n "
+                  + "  * READ_UNCOMMITED\n"
+                  + "  * READ_COMMITED\n"
+                  + "  * REPEATABLE_READ\n"
+                  + "  * SERIALIZABLE\n"
+                  + "  * SQL_SERVER_SNAPSHOT_ISOLATION_MODE\n";
```

Comment:
nit: maybe rename this to `SQL_SERVER_SNAPSHOT` to match other isolation modes that do not have the isolation mode postfix